src/less/donut-graph/donut-graph.less

```diff
@@ -0,0 +1,84 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+@import "../mixins/private/bubble-mixins.less";
+@import "../tooltip/tooltip.less";
```

Comment:
This isn't needed. It will duplicate a bunch of code
Instead add it to here so it generates the correct dependencies: https://github.com/eBay/skin/blob/master/scripts/generate-imports/config.json#L23
Something like:
`'donut-graph': ['tooltip']`

Reply:
@agliga if I remove the import line 63 of this file throws an error as I'm using a tooltip class like a mixin and spreading it's styles into the .donut-graph__tooltip-container class.